pages/glossary/visible-inner-text.md

```diff
@@ -0,0 +1,38 @@
+# Visible inner text
+
+(The "visible inner text" defined here is similar to, but not the same as, [visible text content][].)
+
+The <dfn>visible inner text of a node</dfn> depends on the kind of node. 
+
+The <dfn id="for-text">visible inner text of a [text node][]</dfn> is:
```

Comment:
```suggestion
The <dfn id="visible-inner-text:for-text">visible inner text of a [text node][]</dfn> is:
```
Since definitions are inlined in the final rule pages, it's better to prefix `id` with the definition key (which we know to be unique) to avoid possible problems.